docs/preview/02-Features/sinks/azure-application-insights.md

```diff
@@ -112,6 +128,39 @@ With this configuration, the custom dimension name for the public properties of
 
 > ⚠️ The property format should be in the correct form in order to be used. It's passed to the `String.Format` eventually which will try to insert the exception's property name.
 
+### Correlation
+
+When tracking telemetry, the correlation information is enriched via the [Arcus Serilog correlation enrichment](../telemetry-enrichment.md). 
+This Serilog enricher is capable of customizing the log properties where the correlation will be added. When such custom setup is used, the same custom property names should be used here so that the Application Insights sink knows where to get the correlation information from.
+
+```csharp
+string operationIdPropertyName = "MyOperationPropertyId";
+string transactionIdPropertyName = "MyTransactionPropertyId";
+string operationParentPropertyName = "MyOperationParentPropertyId";
+IServiceProvider serviceProvider = ...
+
+ILogger logger = new LoggerConfiguration()
+    .Enrich.WithCorrelationInfo(serviceProvider, options =>
+    {
+        options.Correlation.OperationIdPropertyName = operationIdPropertyName;
+        options.Correlation.TransactionIdPropertyName = transactionIdPropertyName;
+        options.Correlation.OperationParentIdPropertyName = operationParentIdPropertyName;
+    })
+    .WriteTo.AzureApplicationInsightsWithConnectionString("<connection-string>", options =>
+    {
+        // Sets a custom property name where the operation ID should be added (default: 'OperationId').
+        options.Correlation.OperationIdPropertyName = operationIdPropertyName;
+
+        // Sets a custom property name where the transaction ID should be added (default: 'TransactionId');
+        options.Correlation.TransactionIdPropertyName = transactionIdPropertyName;
+
+        // Sets a custom property name where the operation parent ID should be added (default: 'OperationParentId').
+        options.Correlation.OperationParentIdPropertyName = operationParentIdPropertyName;
+    });
+```
+
+For more information on the way how Arcus handles correlation, see [this dedicated page](../correlation.md).
+
 ## FAQ
 
 ### Q: Why is it mandatory to provide a instrumentation key?
```

Comment:
This is no longer correct, as we also check if an instrumentationkey (or connectionstring) is provided.
At least, we check this in templates if I'm not mistaken.
The question might be a bit misleading.   Maybe it should be:

Why do I have to configure an instrumentationkey (connectionstring) when using the Application Insights sink

Reply:
Done in https://github.com/arcus-azure/arcus.observability/pull/425/commits/aa110d02af14401a143f7b2f8d6740f36e69fba4